reference/promise-types/packages.markdown

```diff
@@ -441,10 +455,28 @@ bundle agent example
   screen-4.1.0-0.19.20120314git3c2946.el7.x86_64
   ```
 
+* Supports [```options```][packages#options] attribute. Each space separate
```

Comment:
This confused me at first. I thought I might need to add a space between a single list item.

When you say "space separate option" do you mean they will eventually become space separated when added to the command lines used in the modules? I think that's what is happening right?

aka

```
EXTERNALURL=https://example.com DEBUG=1 yum update
```